docs/source/api_ref_modules.rst

```diff
@@ -41,7 +42,9 @@ PEFT Components
     peft.AdapterModule
     peft.get_adapter_params
     peft.set_trainable_params
-
+    peft.validate_missing_and_unexpected_for_lora
```

Comment:
Can we register "validate_state_dict_for_lora" in the docstring as a function so it points there?